setup.py

```diff
@@ -46,51 +46,29 @@
 
 try:
     from setuptools._distutils.command.clean import clean as Clean
```

Comment:
```suggestion
    from distutils.command.clean import clean as Clean
```

`setuptools._distutils` is an implementation detail, that is made available through the `distutils` module name.